duck/duckdb.go

```diff
@@ -41,20 +41,25 @@ func NewInMemoryDB(opts ...Opts) DuckDB {
 
 // NewDuckDB creates a new DuckDB
 func NewDuckDB(name string, opts ...Opts) DuckDB {
+	exepath := which.Which("duckdb")
```

Comment:
really should have made this its own PR -- but wanted to make sure the tests still work and not update the hardcoded path